Gaijinizer patch/Map163.txt

```diff
@@ -57,9 +59,9 @@ waiting time.#MTLed
 人間が耐えられる衝撃ではないため、死に至ります。
 > CONTEXT: Map163/events/2/pages/0/27/Dialogue
 > CONTEXT: Map174/events/2/pages/0/27/Dialogue
-[2] do not apply to people. Like Demon rounds, it uses
-magical ore. Because it is not an impact that human beings
-can bear, it leads to death.#MTLed
+[2] Don't fire on humans. Though it uses magical ore as
+its ammunition the same as Demon Rounds, the sheer impact
+of a single round is enough to kill a human.#Ron
 > END STRING
```

Comment:
Minor flow:
>_**Though it uses magical ore in its ammunition
(the same as Demon Rounds), the sheer impact of
a single round is enough to kill a human being.**_